_data/simple-icons.json

```
@@ -2265,6 +2265,11 @@
             "hex": "212121",
             "source": "https://pagekit.com/logo-guide"
         },
+        {
+            "title": "Palantir",
+            "hex": "FFFFFF",
```

I'm not sure if I agree that white (`#FFFFFF`) is their brand colour (especially since white is very generic). Looking at their website and favicon I would expect some kind of dark black/blue colour, e.g. if you scroll down a bit they use `#101113` in their header for text and the logo.

That said, there doesn't seem to be an official source and so this is totally subjective. Let me know if you disagree @mlshv.

```suggestion
            "hex": "101113",
```